test/validators.js

```diff
@@ -6321,6 +6321,22 @@ describe('Validators', () => {
           '353811234567',
         ],
       },
+      {
+        locale: 'en-JM',
+        valid: [
+          '+18762234567',
```

Comment:
You can add some test cases with the separators too. Though my only issue with the separators usually is, will a mix of the separators still be valid? Or could we just do away with the separators all together?

Reply:
@profnandaa we suggested to the other PRs authors to drop separators since it's only aesthetic and to keep the same behaviour for all locales. What do you think?

Reply:
I'm in support 👍